templates/sysv/lsb-3.1/init.sh

```diff
@@ -88,7 +88,13 @@ stop() {
       sleep 1
     done
     if status ; then
-      emit "$name stop failed; still running."
+      emit "$name stop failed; still running.  Killing with SIGKILL"
+      trace "Killing $name (pid $pid) with SIGKILL"
+      kill -KILL $pid
```

Comment:
I'd prefer if this were optional, with the default being to not send SIGKILL

Maybe a setting `KILL_ON_STOP_TIMEOUT` being set would determine whether SIGKILL or just logging "Stop failed ... still running" was used.